src/tools/ScenarioMeasurement/Util/ProcessHelper.cs

```diff
@@ -265,7 +265,7 @@ public ProcessResult Run()
                 Thread.Sleep(MeasurementDelay * 1000);
                 if (!process.HasExited)
                 {
-                    if(GuiApp)
+                    if (GuiApp)
```

Comment:
As I mentioned offline, a better way to measure the startup is to ready the console output from PDN and remove the code that we have today for `AppStarted`, `AppIdle`, etc.